manala.backup-manager/templates/configs/mysql.j2

```diff
@@ -446,21 +446,21 @@ export BM_PIPE_COMPRESS
 # account. See http://aws.amazon.com
 
 # The bucket to upload to. This bucket must be dedicated to backup-manager
-export BM_UPLOAD_S3_DESTINATION=""
+{{ macros.config_row(config, 'BM_UPLOAD_S3_DESTINATION', '') }}
 
 # the S3 access key provided to you
-export BM_UPLOAD_S3_ACCESS_KEY=""
+{{ macros.config_row(config, 'BM_UPLOAD_S3_ACCESS_KEY', '') }}
 
 # the S3 secret key provided to you
-export BM_UPLOAD_S3_SECRET_KEY=""
+{{ macros.config_row(config, 'BM_UPLOAD_S3_SECRET_KEY', '') }}
 
 # purge archives on remote hosts before uploading?
-export BM_UPLOAD_S3_PURGE="false"
+{{ macros.config_row(config, 'BM_UPLOAD_S3_PURGE', 'false') }}
```

Comment:
You can directly make usage of booleans as they are well handled by the config macro.
`'false'`-> `false`